src/plugins/presentation_panel/public/panel_component/panel_header/presentation_panel_hover_actions.tsx

```diff
@@ -462,15 +470,65 @@ export const PresentationPanelHoverActions = ({
         /\s/g,
         ''
       )}`}
+      css={css`
+        border-radius: ${euiThemeVars.euiBorderRadius};
+        position: relative;
+        height: 100%;
+
+        .embPanel {
+          ${showBorder ? `outline: ${viewMode === 'edit' ? DASHED_OUTLINE : SOLID_OUTLINE};` : ''}
```

Comment:
**nit** The [style guide](https://docs.elastic.dev/kibana-dev-docs/contributing/styleguide#only-use-ternary-operators-for-small-simple-code) suggests we should avoid nested ternaries

Reply:
Does it count as a nested ternary if it's a ternary inside of the CSS template string?

Maybe I can add some extra lines to make it easier to read like this?:

```suggestion
          ${showBorder ? `
            outline: ${viewMode === 'edit' ? DASHED_OUTLINE : SOLID_OUTLINE};
          ` : ''}
```

Reply:
I personally found it to be a bit confusing to read at first, which is why I brought it up - I had to do a double take when I first saw it 😆 I like the extra spacing - that makes it much easier IMO :+1: